packages/outline-templates/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@phase2/outline-templates",
   "description": "The Outline Starter Templates",
-  "version": "0.0.33",
+  "version": "0.0.34",
```

Comment:
Should I be changing these or is this automatically managed on merging based on the conventional commit messages?

Reply:
Yes, because `outline-templates` had content updated (the `package.json` in the `default` directory, this should be bumped an a new release created for the template.

Reply:
I need to document (and automate) the Changeset releases to make this easier to understand.